src/docs/release/breaking-changes/eliminating-nullok-parameters.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Eliminating `nullOk` Parameters
+description: To eliminate `nullOk` parameters to help with API sanity in the face of null safety.
+---
+
+## Summary
+
+This migration guide describes conversion of code that uses the `nullOk`
+parameter on multiple `of` static accessors and related accessors to use
+alternate APIs with nullable return values.
+
+## Context
+
+Flutter has a common pattern of allowing lookup of some types of widgets
+(`InheritedWidget`s) using static member functions that are typically called
+`of`, and take a `BuildContext`.
+
+Before non-nullability was the default, it was useful to have a toggle on these
+APIs that swapped between throwing an exception if the widget was not present in
+the widget tree and returning null if it was not found. It was useful, and
+wasn't confusing, since every variable was nullable.
+
+When non-nullability was made the default, it was then desirable to have the
```

Comment:
nit: maybe break this paragraph up into multiple sentences to improve readability?

Reply:
Run-on sentence much? Yeah, I split it into multiple sentences and multiple paragraphs.